### PR TITLE
GH-459: navbar

### PR DESCRIPTION
**Pull request by Baeck016**
Done with navbar

### CODE REVIEW

File: ui/src/components/NavigationBar/MainMenu.js [hunk 1]
```diff
@@ -173,9 +172,17 @@ const MainMenu = ({ login, onClose }) => {
...
                     </LinkOverlay>
                   </Button>
+                  <Button
+                    onClick={onClose}
+                    marginBottom="5"
+                    width="100%"
+                    style={{ WebkitMarginStart: '0', marginInlineStart: '0' }}
+                  >
+                    <LinkOverlay href="/signup">SIGN UP</LinkOverlay>
+                  </Button>
```
Comment (bhryck) [inline ui/src/components/NavigationBar/MainMenu.js:185]: image
the signup button should be the same color as the navbar when not hovered on.

Comment (bhryck) [reply ui/src/components/NavigationBar/MainMenu.js:185]: actually saw diff versions on the figma, if that the one cxo decided on then no need to change

File: ui/src/components/NavigationBar/MainMenu.js [hunk 2]
```diff
@@ -137,17 +137,16 @@ const MainMenu = ({ login, onClose }) => {
                 >
```
Comment (bhryck) [inline ui/src/components/NavigationBar/MainMenu.js:137]: image
image

color of the bg looks different, may need to check with cxo if they want to change the bg image color

File: ui/src/components/NavigationBar/MainMenu.js (hunk 1 above)
Comment (Baeck016) [reply ui/src/components/NavigationBar/MainMenu.js:185]: > actually saw diff versions on the figma, if that the one cxo decided on then no need to change

It's the one top, the one with blue and dark blue when hovered on.

File: package.json [hunk 3]
```diff
@@ -5,6 +5,10 @@
     "prettier": "2.3.2"
   },
   "dependencies": {
+    "ag-grid-community": "^28.0.2",
```
Comment (albertparedandan) [inline package.json:8]: this isn't supposed to be here. please remove

**Review by albertparedandan — CHANGES_REQUESTED** [review]

File: ui/src/components/NavigationBar/MainMenu.js [hunk 4]
```diff
@@ -137,17 +137,16 @@ const MainMenu = ({ login, onClose }) => {
...
                   <Button marginBottom="5" width="100%">
                     <LinkOverlay href="/profile" onClick={onClose}>
-                      My Profile
+                      MY PROFILE
```
Comment (albertparedandan) [inline ui/src/components/NavigationBar/MainMenu.js:140]: all these buttons do not work when i press the actual button itself. have to click the text.

**Review by albertparedandan — APPROVED** [review]

**Pull request merged**